src/_ert/forward_model_runner/reporting/event.py

```diff
@@ -137,6 +137,7 @@ async def handle_publish(self, client: Client) -> None:
                 return
             except ClientConnectionError as exc:
                 logger.error(f"Failed to send event: {exc}")
+                self._reporter_exception = exc
```

Comment:
But this can be a temporal drop out, so why to bring down reporter instead of reconnecting?